config/env.ini

```diff
@@ -42,6 +42,9 @@ SPC_CONCURRENCY=${CPU_COUNT}
 SPC_SKIP_PHP_VERSION_CHECK="no"
 ; Ignore some check item for bin/spc doctor command, comma separated (e.g. SPC_SKIP_DOCTOR_CHECK_ITEMS="if homebrew has installed")
 SPC_SKIP_DOCTOR_CHECK_ITEMS=""
+; extra modules that xcaddy will include in the FrankenPHP build
+SPC_CMD_VAR_FRANKENPHP_XCADDY_MODULES="--with github.com/dunglas/frankenphp/caddy --with github.com/dunglas/mercure/caddy --with github.com/dunglas/vulcain/caddy --with github.com/dunglas/caddy-cbrotli"
```

Comment:
Are these modules required all the time? If so, it would be better to add them as dependencies directly.

Reply:
(also, again, can't download them earlier as it's part of xcaddy's process)